### DISCUSSION TITLE
Download failed, intermittent success and manytimes failed. Is it because limitation??

### DISCUSSION THREAD

**Discussion by sbudhi** [aclap-dev/video-downloadhelper#1478] [Help and Q&A]
### Your question

I've purchase this apps in term to unlock limitation but after purchase license it's success for 2-3days after that it's back bugging me with "Download failed, Sadly we failed at downloading that specific media..." moreover when I put more than 1 queue it's only success 1 and other failed, when I tried that time keep failed (but when I played video online those video have no issue, it's played perfectly fine).
I tried about 5-15 minutes later can continue but only success 1 video and other keep failed..
Is this came from limitation? If that so I request refund because the term before, purchase license is to lift up limitation..

### Addon & CoApp version and details

I've purchase this apps in term to unlock limitation but after purchase license it's success for 2-3days after that it's back bugging me with "Download failed, Sadly we failed at downloading that specific media..." moreover when I put more than 1 queue it's only success 1 and other failed, when I tried that time keep failed (but when I played video online those video have no issue, it's played perfectly fine).
I tried about 5-15 minutes later can continue but only success 1 video and other keep failed..
Is this came from limitation? If that so I request refund because the term before, purchase license is to lift up limitation..

**Comment by Mattmjs** [comment]
There is no limitation when you have a license.
If you provide your version information for the
software and the link where the problem happens 
it can be investigated further.

**Reply by sbudhi** [reply]
Version 9.1.0.44 (stable), target mozilla, locale en-US.
Companion App installed (v2.0.19)

**Reply by Mattmjs** [reply]
And a video where the problem happens ? otherwise no one knows what the issue is.

**Reply by sbudhi** [reply]
![videodownloader-failed-01](https://github.com/user-attachments/assets/c7634a90-4ccf-4789-8bf7-20f7f421ad3c)

**Reply by sbudhi** [reply]
when I clicked "Report" does it have have ticket to follow-up?

**Reply by Mattmjs** [reply]
What it does is notifies the developer of the problem site or video and implement a fix if possible. If you've done that already then
it will be looked into but I don't think you get notified on the progress of it. There are some exceptions where there couldn't be a fix.
Such as DRM protected video or encrypted video. If you look up near the address in Firefox you should see this icon for DRM.
Maybe that is your problem or it could be something else.

![DRM icon](https://github.com/user-attachments/assets/d6287f28-e432-41d2-9397-d74593aacd02)
